<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cultura</class>
 <widget class="QDialog" name="Cultura">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Se alege din lista urmatoare liderul:</string>
   </property>
  </widget>
  <widget class="QLabel" name="decizie">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Prima initiativa:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>691</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="l2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>A doua initiativa:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>691</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="decizie2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Daca doriti modificarea acestora, completati campurile de mai jos.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="salveaza">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>410</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Salveaza si inchide</string>
   </property>
  </widget>
  <widget class="QPushButton" name="instintare">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>410</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Instinteaza liderul </string>
   </property>
  </widget>
  <widget class="QPushButton" name="salveaza2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>450</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Salveaza</string>
   </property>
  </widget>
  <widget class="QLabel" name="i1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>761</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="i2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>761</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
